mne/io/write.py

```diff
@@ -411,7 +411,7 @@ def _generate_meas_id():
     id_ = dict()
     id_['version'] = FIFF.FIFFC_VERSION
     id_['machid'] = get_machid()
-    id_['secs'], id_['usecs'] = _date_now()
+    id_['secs'], id_['usecs'] = np.array([0, 0], dtype='int32')
```

Comment:
you prefer to put 0 rather than omitting to write the tag?

Reply:
IIRC we agreed to not write the tag, set as `None` when reading if not present, and in functions that require a `meas_date` (like annotations) treat `None` as the same as `[0, 0]`

Reply:
I think this tag cannot be omitted. C.f., also the anonymization discussions we had with Matti.

Reply:
I don't think this is where we should change this. This will set `meas_id` to `(0, 0)` for all newly written files.

Reply:
okay I see. Let me see if I can come up with a better fix

Reply:
actually I am not sure anymore of the right way to do it. We have to write an ID tag in the fif file and this has to contain 8 bytes for the date. @larsoner what do you think it the right approach?

Reply:
IIUC we currently "time stamp" the file creation time. How does this break BIDS / why do they forbid this?

If there is a good reason, maybe we should abandon this practice. Using `(0, 0)` effectively does this.